runtimes/runtimes/standalone.ts

```diff
@@ -1,4 +1,4 @@
-import { TextDocuments } from 'vscode-languageserver'
+import { DidChangeWorkspaceFoldersNotification, TextDocuments } from 'vscode-languageserver'
```

Comment:
nit: after refactoring project structure, it's better to import protocol objects from `../protocol` subfolder, as we use it as source of truth for protocol now.

Reply:
Fixed